src/Storefront/Resources/app/storefront/src/plugin/clear-input-button/clear-input.plugin.js

```diff
@@ -17,9 +16,7 @@ export default class ClearInputPlugin extends Plugin {
     };
 
     init() {
-        this.clearButtons = DomAccess.querySelectorAll(
-            document,
-            this.options.clearButtonSelector
+        this.clearButtons = document.querySelectorAll(this.options.clearButtonSelector
```

Comment:
Not important, but maybe move line 20 up here. 

```suggestion
        this.clearButtons = document.querySelectorAll(this.options.clearButtonSelector);
```

Reply:
Fixed.